### ISSUE TITLE
usage

### ISSUE THREAD

**Issue by mfazel** [mikkoch/unicorn-qc#2]
Very nice pipeline, but how is it being used? 
Any example or help for the modules if this was meant to support your paper?
Thanks

**Comment by FunGeST** [comment]
Hi.. did you received any answer. You right this is pity that no examples or usages description is provided
Thanks

**Comment by mfazel** [comment]
No, Unfortunately it seemed very appealing and nice pipeline, but when you look at the repository activity, you realize that this has been just to satisfy the journal requirement. Another unanswered issue is there too. I used other methods since this wasn't clear and didn't work for me.

**Comment by EasperOne** [comment]
Yeah a little disappointed here. The script doesn't quite match what they described in the paper. For example they used 6 PCs instead of 20 like in the paper. Does anyone knows interesting pipelines for cohort integration in GWAS?

**Comment by nandobonf** [comment]
I came across this after reading the interesting HMG paper, but it’s really disappointing to see how difficult it is to actually apply it. Quite surprising, especially with names like B. Neale among the authors. @mikkoch @nealelab
